.changeset/unlucky-ravens-type.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
+---
+
+The property `compressHTML` is now `true` by default.
```

Comment:
``````suggestion
The property `compressHTML` is now `true` by default. Setting this value to `true` is no longer required.

If you do not want to minify your HTML output, you must set this value to `false` in `astro.config.mjs`.

```js
{
  compressHTML: false
}
```
``````
Especially since this is a change that reverses the behaviour of many existing projects, it is particularly important to provide guidance as to what people should *do* in response to this change in default to keep their original settings. 

(In general, it's *always* good to include any actions people might need to take in the changesets! This is where people will look to see what has changed, and what they might need to worry about, and docs only ever shows "how things are now.")